articles/Silo_Finance.md

```diff
@@ -0,0 +1,448 @@
+
+
+
+<p id="gdcalert4" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image4.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert5">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image4.png "image_tooltip")
+
+
+(source: [SiloLens.sol](https://etherscan.io/address/0xf12C3758c1eC393704f0Db8537ef7F57368D92Ea#code))
+
+
+
+* **[Silo ](https://etherscan.io/address/0xd998C35B7900b344bbBe6555cc11576942Cf309d#code)**is the main component of the protocol that acts as a vault for assets, implementing the lending logic, managing and isolating the risk, and performing liquidations.
+* **[PriceProviderRepository](https://etherscan.io/address/0x7C2ca9D502f2409BeceAfa68E97a176Ff805029F#code)** manages the oracle modules and the price request routing for each silo. It can support many protocols and sources. Currently, it supports three oracle sources: [Chainlink](https://etherscan.io/address/0xe37B8c83138caF12E57632D19c06Eb561D47e423#code) (default price provider), UniswapV3 TWAP, and BalancerV2 TWAP.
+* **[Interest Rate Model](https://etherscan.io/address/0x7e9e7ea94e1ff36e216a703d6d66ece356a5fd44#code)**. Silo Finance uses a dynamic interest rates model which is described in more detail in this [Interest Rate Model paper](https://drive.google.com/file/d/11jzbdIQ9KGm_ZVFIUwUu44O1ktWx5R30/view?pli=1).
+* **[Silo Router](https://etherscan.io/address/0xb2374f84b3cEeFF6492943Df613C9BcF45322a0c#code) **is a utility contract that can batch any number or combination of actions (Deposit, Withdraw, Borrow, Repay) and execute in a single transaction.
+* The permission system registry consists of three smart contracts: GuardedLaunch.sol, TwoStepOwnable.sol, and Manageable.sol
+
+
+
+<p id="gdcalert5" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image5.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert6">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+
+![alt_text](images/image5.png "image_tooltip")
+
+
+(source: [Certora Formal Verification Document](https://www.silo.finance/_files/ugd/fd5034_98066b4c70ea46b5862a00670179f067.pdf))
+
+
+# SILO Governance
+
+The governance process follows a standard procedure of forum discussion, Snapshot vote, and finally on-chain vote. For more details visit the [docs](https://silopedia.silo.finance/governance/silodao). In practice, however, Snapshot is mostly applied for decisions concerning expenditures of the DAO’s funds, whereas changes that affect the protocol itself are directly voted on via an on-chain proposal on [Tally](https://www.tally.xyz/governance/eip155:1:0xA89163F7B2D68A8fbA6Ca36BEEd32Bd4f3EeAf61).
```

Comment:
`Snapshot is mostly applied for decisions concerning expenditures of the DAO’s funds` Looks like because treasury funds managed by a 2/3 multisig. The address contains ~8m USDC, ~3m UNIv3 LPs, ~260k vlCVX, and some additional funds deployed to their silos.

Reply:
yep we forgot to mention the multisig in the text, it was only in the sources. But added it now